### PR TITLE
[PHP-6] add class Arrays

### PR DESCRIPTION
**Pull request by oleksiivv**

### CODE REVIEW

File: src/arrays/Arrays.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    /**
+     * @param array $input
+     * @return array
+     */
+    public function repeatArrayValues(array $input): array
+    {
+        $result = [];
+        $arrayCount = count($input);
+
+        for ($i = 0; $i < $arrayCount; $i++) {
```
Comment (yderevianko-cartera) [inline src/arrays/Arrays.php:20]: could you please try to use php array functions?

File: src/arrays/Arrays.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    public function getUniqueValue(array $input): int
+    {
+        $uniqueValue = 0;
+        $uniqueValues = [];
+
+        if (count($input) === 0) {
+            return $uniqueValue;
+        }
+
+        foreach ($input as $item) {
```
Comment (yderevianko-cartera) [inline src/arrays/Arrays.php:42]: could you please try to use php array functions?

**Review by yderevianko-cartera — CHANGES_REQUESTED** [review]

File: src/arrays/Arrays.php (hunk 1 above)
Comment (oleksiivv) [reply src/arrays/Arrays.php:20]: done

File: src/arrays/Arrays.php (hunk 2 above)
Comment (oleksiivv) [reply src/arrays/Arrays.php]: done

**Review by yderevianko-cartera — APPROVED** [review]

**Pull request merged**